lib/node_modules/@stdlib/lapack/base/dlaswp/docs/types/index.d.ts

```diff
@@ -0,0 +1,119 @@
+/*
+* @license Apache-2.0
+*
+* Copyright (c) 2024 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+// TypeScript Version: 4.1
+
+/// <reference types="@stdlib/types"/>
+
+import { Layout } from '@stdlib/types/blas';
```

Comment:
```suggestion
import { Layout } from '@stdlib/types/blas';
```

Ideally we should import from `@stdlib/types/lapack`, but as we have no other files in namespace I kept it. Not sure if this is correct.

Reply:
This is fine for now. It will be a minor refactoring once we add LAPACK-specific type definitions.